go.sum

```diff
@@ -283,6 +283,8 @@ github.com/getlantern/filepersist v0.0.0-20210901195658-ed29a1cb0b7c/go.mod h1:8
 github.com/getlantern/flashlight v0.0.0-20230403092335-5f84ae10c585 h1:lVu4HyDAxXkX4kJEJQWpJJz+Poprb0crUGW5zhoh3GU=
 github.com/getlantern/flashlight/v7 v7.6.71 h1:/IYezK1+PSClDC9V/3JgNs9UqkHP96h+CiR9JkMlls0=
 github.com/getlantern/flashlight/v7 v7.6.71/go.mod h1:n5EO3xn22j9uzRxyalP0G9TticdfWD26r6JEkfkSShE=
+github.com/getlantern/flashlight/v7 v7.6.72-0.20240411074118-1230c1415326 h1:ULia/RZxAgPiCr/KDcts2UiBGq9si5KKQi5pBLSOwjo=
```

Comment:
I can't find commit `1230c1415326` on any branch of the flashlight repo. Did you push these changes?

I guess we can just update to v7.6.72 now that https://github.com/getlantern/flashlight/pull/1370 is merged. Let me do that..

Reply:
Thanks for updating flashlight.